src/index.html

```diff
@@ -17,16 +17,30 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-form"
+      action="#"
+      data-qa="big"
+    >
```

Comment:
The attributes in the <form> tag should start on a new line with a 2-space indentation relative to the tag for better readability as per the checklist.